src/helix-ui/styles/reset.less

```diff
@@ -25,7 +25,6 @@ html {
   box-sizing: inherit;
 }
 
-@import 'reset/forms';
```

Comment:
* moved valid resets to `components/forms.less`
* all other resets removed (either useless or duplicate)